examples/echo/main.c

```diff
@@ -143,12 +144,30 @@ static void handle_rx (YggWorker   *worker,
   g_bytes_unref (data);
 }
 
+static gboolean
+sigint_callback(gpointer main_loop) {
```

Comment:
Since this worker is an example, could we add a comment here hinting to developers that this is a good place to perform any resource clean up before the application terminates. (I assume this is why you have added a SIGINT handler.)

Reply:
Comment added.